system-tests/test/issue_8280_spec.js

```diff
@@ -0,0 +1,91 @@
+const { fs } = require('@packages/server/lib/util/fs')
+const Fixtures = require('../lib/fixtures')
+const systemTests = require('../lib/system-tests').default
+
+const PROJECT_NAME = 'issue-8280-retain-video'
+
+describe('e2e issue 8280', () => {
+  systemTests.setup()
+
+  // https://github.com/cypress-io/cypress/issues/8280
+
+  it('should retain the videos from previous runs if trashAssetsBeforeRuns=false', function () {
```

Comment:
I think this can be async/await? It will make the test easier to read

Reply:
converted to async/await :+1: